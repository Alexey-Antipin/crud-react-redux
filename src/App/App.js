import { BrowserRouter, Routes, Route } from "react-router-dom";
import { Navbar } from "../components/navbar/Navbar";
import { MainPage } from "../components/page/MainPage/MainPage";
import { Page } from "../components/page/MainPage/Page";
import "../App/App.scss";

function App() {
  return (
    <div className="App">
      <div className="Apps">
        <BrowserRouter>
          <Navbar />
          <Routes>
            <Route path="/" element={<MainPage />} />
            <Route path="/PageTodo" element={<Page />} />
          </Routes>
        </BrowserRouter>
      </div>
    </div>
  );
}

export default App;
