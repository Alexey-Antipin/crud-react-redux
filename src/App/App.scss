* {
    margin: 0;
    padding: 0;
}

a {
    text-decoration: none;
    color: black;
}

body {
    background-attachment: fixed;
    background-color: rgb(201, 206, 206);
    background-image: url("../img/loules.jpg");
    background-repeat: no-repeat;
    background-size: 100% 100%;
}