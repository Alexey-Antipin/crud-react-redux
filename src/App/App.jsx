import { BrowserRouter, Routes, Route } from "react-router-dom";
import { Navbar } from "../components/navbar/Navbar";
import { MainPage } from "../components/page/MainPage/MainPage";
import { Page } from "../components/page/MainPage/Page";
import "./App.scss";

export const App = () => {
  return (
    <BrowserRouter>
      <Navbar />
      <Routes>
        <Route path="/" element={<MainPage />} />
        <Route path="/PageTodo" element={<Page />} />
      </Routes>
    </BrowserRouter>
  );
};
