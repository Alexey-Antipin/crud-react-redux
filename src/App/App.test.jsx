import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { App } from "./App";

describe("App", () => {
  it("App snapshot", () => {
    const { asFragment } = render(<App />);
    expect(asFragment(<App />)).toMatchSnapshot();
  });
});
