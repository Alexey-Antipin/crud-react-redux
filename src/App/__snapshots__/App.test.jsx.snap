// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="Apps"
    >
      <div
        class="Navbar"
      >
        <header>
          <button
            class="Navbar__button"
          >
            <a
              href="/PageTodo"
            >
              Список todo
            </a>
          </button>
        </header>
      </div>
      <div
        class="Logo"
      >
        <div
          class="Logo__animation"
        >
          <div
            class="Logo__logo_up"
          >
            <img
              alt="logo"
              class="Logo__logo_down"
              src="sches.svg"
            />
          </div>
        </div>
        <div
          class="Logo__text"
        >
          <h3>
            Добро пожаловать
          </h3>
          на главную страницу, чтобы перейти 
          <br />
          к Todo листу, наверху есть кнопка,
          <br />
          кликните по ней "Список todo".
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
